File: Lab3/src/main/java/domain/Proba.java
package domain;

public class Proba extends Entity<Long> {
    private String distanta;
    private String stil;

    public Proba() {}

    public Proba(String distanta, String stil) {
        this.distanta = distanta;
        this.stil = stil;
    }

    public String getDistanta() {
        return distanta;
    }

    public void setDistanta(String distanta) {
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }
}

File: Lab3C#/domain/Proba.cs
namespace Lab3.domain;

public class Proba : Entity<long>
{
    public string Distanta { get; set; }
    public string Stil { get; set; }

    public Proba() { }

    public Proba(string distanta, string stil)
    {
        Distanta = distanta;
        Stil = stil;
    }
}